scripts/Units/armsnipe.bos

```diff
@@ -823,7 +824,7 @@ lua_UnitScriptLight(lightIndex, count)
 
 SetMaxReloadTime(Func_Var_1)
 {
-	restore_delay = 60000; //60000
+	// stub
```

Comment:
Can this empty function be deleted? If not and an empty stub is required, please change the comment to explain why this is necessary.

Reply:
function deleted